gslib/commands/cp.py

```diff
@@ -81,194 +81,188 @@
 
 _DESCRIPTION_TEXT = """
 <B>DESCRIPTION</B>
-  The gsutil cp command allows you to copy data between your local file
-  system and the cloud, copy data within the cloud, and copy data between
+  The ``gsutil cp`` command allows you to copy data between your local file
+  system and the cloud, within the cloud, and between
   cloud storage providers. For example, to upload all text files from the
-  local directory to a bucket you could do:
+  local directory to a bucket, you can run:
 
     gsutil cp *.txt gs://my-bucket
 
-  Similarly, you can download text files from a bucket by doing:
+  You can also download text files from a bucket:
 
     gsutil cp gs://my-bucket/*.txt .
 
-  If you want to copy an entire directory tree you need to use the -r option.
-  For example, to upload the directory tree "dir":
+  Use the ``-r`` option to copy an entire directory tree.
+  For example, to upload the directory tree ``dir``:
 
     gsutil cp -r dir gs://my-bucket
 
-  If you have a large number of files to transfer you might want to use the
-  top-level gsutil -m option (see "gsutil help options"), to perform a
-  parallel (multi-threaded/multi-processing) copy:
+  If you have a large number of files to transfer, you can use the
+  top-level gsutil ``-m`` option to perform a
+  parallel multi-threaded/multi-processing copy:
 
     gsutil -m cp -r dir gs://my-bucket
 
-  You can pass a list of URLs (one per line) to copy on stdin instead of as
-  command line arguments by using the -I option. This allows you to use gsutil
-  in a pipeline to upload or download files / objects as generated by a program,
-  such as:
+  You can use the ``-I`` option with ``stdin`` to specify a list of URLs to
+  copy, one per line. This allows you to use gsutil
+  in a pipeline to upload or download objects as generated by a program:
 
-    some_program | gsutil -m cp -I gs://my-bucket
+    cat filelist | gsutil -m cp -I gs://my-bucket
 
   or:
 
-    some_program | gsutil -m cp -I ./download_dir
+    cat filelist | gsutil -m cp -I ./download_dir
 
-  The contents of stdin can name files, cloud URLs, and wildcards of files
-  and cloud URLs.
+  where the output of ``cat filelist`` is a list of files, cloud URLs, and
+  wildcards of files and cloud URLs.
 
-  NOTE: Shells (like bash, zsh) sometimes attempt to expand wildcards in ways
-  that can be surprising. Also, attempting to copy files whose names contain
-  wildcard characters can result in problems. For more details about these
-  issues see the section "POTENTIALLY SURPRISING BEHAVIOR WHEN USING WILDCARDS"
-  under "gsutil help wildcards".
+  NOTE: Shells like ``bash`` and ``zsh`` sometimes attempt to expand
+  wildcards in ways that can be surprising. You may also encounter issues when
+  attempting to copy files whose names contain wildcard characters. For more
+  details about these issues, see "Potentially Surprising Behavior When Using Wildcards"
+  under ``gsutil help wildcards``.
```

Comment:
I believe keeping these in quotations (instead of ``) will render the contents as a link?